uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/audit/AuditCheckMockMvcTests.java

```diff
@@ -986,7 +987,6 @@ void generateUserCreatedEvent_DuringLoginServerAuthorize() throws Exception {
     }
 
     @Nested
-    @DefaultTestContext
```

Comment:
This annotation creates another context, and this new context doesn't have the required listeners